kong/plugins/key-auth/migrations/002_add_signature_validation.lua

```diff
@@ -0,0 +1,15 @@
+return {
+  postgres = {
+    up = [[
+      ALTER TABLE IF EXISTS ONLY "keyauth_credentials"
+        ADD COLUMN "secret" TEXT;
+    ]],
+  },
+
+  cassandra = {
+    up = [[
+      ALTER TABLE IF EXISTS ONLY "keyauth_credentials"
+        ADD COLUMN "secret" text;
```

Comment:
This is not a valid CQL statement, this migration will not run in Cassandra (hence the importance of integration tests in PRs!). `ALTER TABLE IF EXISTS` is not supported.